tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -266,7 +266,7 @@ public function updateResourceShare(string $user, TableNode  $body, string $perm
 		}
 
 		if (\array_key_exists('expirationDateTime', $bodyRows)) {
-			$body["expirationDateTime"] = $bodyRows['expirationDateTime'];
+			$body["expirationDateTime"] = empty($bodyRows["expirationDateTime"]) ? null : $bodyRows["expirationDateTime"];
```

Comment:
```suggestion
			$body['expirationDateTime'] = empty($bodyRows['expirationDateTime']) ? null : $bodyRows['expirationDateTime'];
```
Let's make a uniform.